rocketpy/Components.py

```diff
@@ -0,0 +1,59 @@
+__author__ = "Mateus Stano Junqueira"
+__copyright__ = "Copyright 20XX, RocketPy Team"
+__license__ = "MIT"
+
+from collections import namedtuple
+
+
+class Components:
+    """A group of components, each of which has a position"""
+
+    def __init__(self):
+        self.component_position = namedtuple("component_tuple", "component position")
```

Comment:
The namedtuple has two parameters: `component` and `position`. I changed the name of the variable to be `self.component_tuple` now

This actually comes from a different implementation of the Components class that I was trying out. Here there is no practical utility besides making it easier to understand and use (at least I think so). 

It could just be a tuple but I find a namedtuple to work great since it essentially works as a tuple (you can get the position of a component with index, e.g. `Calisto.aerodynamicSurfaces[0][0]` or with the attr name, e.g. `Callisto.aerodynamicSurfaces[0].position`)

My initial intention was that this namedtuple would be called just `component`, and the class `Components` would be a holder of `component`'s. I changed the name because this happened:

![image](https://github.com/RocketPy-Team/RocketPy/assets/69485049/ca5a373d-4bc4-4471-94b8-ae2550238b88)

I just didn't like how the word `component` repeats itself.

I am okay with the changing the name or just removing the namedtuple altogether. What is you suggestion?

Reply:
I think with the new "component_tuple" name we are already in a better situation than in the figure above.

Naming the tuples argument seems more convenient to me when reading.